map/change-basemap/src/main/java/com/esri/samples/change_basemap/ChangeBasemapSample.java

```diff
@@ -53,27 +51,32 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
       // creates a map view
       mapView = new MapView();
 
-      // setup listview of basemaps
-      ListView<Basemap.Type> basemapList = new ListView<>(FXCollections.observableArrayList(Basemap.Type.values()));
-      basemapList.setMaxSize(250, 150);
+      // setup a listview of basemaps styles
```

Comment:
Could do plain english here as well?
```suggestion
      // setup a list view of basemaps styles
```